ui/printer.py

```diff
@@ -117,7 +118,34 @@ def exit_printer():
         i.set_callback("KEY_LEFT", exit_printer)
         i.set_callback("KEY_ENTER", exit_printer)
         i.listen()
+    image_width, image_height = image.size
+    size = o.width, o.height
+    width = o.width/image_width
+    height = o.height/image_height
+    if height < width:
+        multiplier = height
+    else:
+        multiplier = width
+    multiplier = trunc(multiplier)
```

Comment:
If you do integer division, is it even necessary to use `trunc`? Otherwise, maybe you need to do float division in L123 and L124?